packages/nuxt/build.config.ts

```diff
@@ -27,6 +27,7 @@ export default defineBuildConfig({
     'ofetch'
   ],
   externals: [
+    'consola',
```

Comment:
The build failed with the error: 
> │ - Inlined implicit external consola
> │  ERROR  Exiting with code (1). You can change this behavior by setting failOnWarn: false .

I wasn't sure whether I should add consola here or set `failOnWarn: false` to the build config to solve this.

Reply:
instead we should add `consola` as a dependency